src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$day-size: 100px;
+$week: 7;
+$gap: 1px;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  display: flex;
+  width: 100%;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: 707px;
```

Comment:
The width of the `.calendar` should not be hardcoded to `707px`. Instead, it should be calculated based on the `$day-size`, `$week`, and `$gap` variables to ensure the calendar is responsive to changes in these values.